packages/venia-concept/src/components/ProductOptions/option.js

```diff
@@ -54,6 +69,9 @@ class Option extends Component {
                     items={values}
                     onSelectionChange={handleSelectionChange}
                 />
+                <p className={classes.selection}>
+                    {selectedValue?`Selected ${label} : ${this.valuesMap.get(Array.from(selectedValue).pop())}`:''}
```

Comment:
I think this would be more readable if you pulled it out above the `return`.

For example:
```js
const selectedLabel = selectedValue ? ... : '';

return (
  ...
   <p className={classes.selection>
      {selectedLabel}
  </p>
  ...
);
```

Reply:
sure will make that change. :+1: